homeassistant/components/lcn/__init__.py

```diff
@@ -168,8 +175,8 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
         host = hass.data[DOMAIN].pop(config_entry.entry_id)
         await host[CONNECTION].async_close()
 
-    # unregister service calls
-    if unload_ok and not hass.data[DOMAIN]:  # check if this is the last entry to unload
+    # unregister service calls if this is the last entry to unload
```

Comment:
Services should not be removed
They should be always available, and raise if they are unable to run (eg. no entry loaded)